zeebe/bpmn-model/src/test/java/io/camunda/zeebe/model/bpmn/instance/zeebe/ZeebeCalledDecisionTest.java

```diff
@@ -41,4 +48,39 @@ public Collection<AttributeAssumption> getAttributesAssumptions() {
         new AttributeAssumption(
             BpmnModelConstants.ZEEBE_NS, "bindingType", false, false, ZeebeBindingType.latest));
   }
+
+  @Test
+  public void shouldReadValidBindingType() {
+    // given
+    final BpmnModelInstance modelInstance =
+        Bpmn.readModelFromStream(getClass().getResourceAsStream("ZeebeCalledDecisionTest.bpmn"));
```

Comment:
🔧 From the name of the test method and from the name of the bpmn resource it's unclear that this verifies that the binding type of the called decision is `deployment`. My suggestion is to adjust the bpmn filename.

🔧 I'm not a big fan of bpmn files in tests, as they make the tests harder to understand (the information is split over multiple places), like above. We could use the bpmn-model api to create the bpmn model, produce the xml, and then read it again all in the same test case to avoid this.

Reply:
Thanks, good points! I replaced the BPMN files with models created by the API for both cases (valid and invalid binding type).